Cargo.toml

```diff
@@ -311,12 +311,21 @@ strip = false
 debug = "full"
 lto = false
 
+# Profile for fast test execution: Skip debug info generation, and
+# apply basic optimization, which speed up build and running tests.
 [profile.fast-build]
```

Comment:
If it's oriented for tests should we call it `profile.testing`? or similar?

Reply:
Yes, I only wanted to avoid the churn here.